mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -2291,4 +2297,66 @@
+    <string name="automatic_translation_option_never_translate_summary_preference">%1$s will never offer to translate sites in this language.</string>
+
+    <!-- Never translate site preference screen -->
+    <!-- Title of the never translate site preference screen that will appear on the toolbar.-->
+    <string name="never_translate_site_toolbar_title_preference">Never translate these sites</string>
+    <!-- Screen header presenting the never translate site preference feature. It will appear under the toolbar. -->
+    <string name="never_translate_site_header_preference">To add a new site: Visit it and select “Never translate this site” from the translation menu.</string>
+    <!-- Content description (not visible, for screen readers etc.): For a never-translated site list item that is selected.
+             The first parameter is web site url (for example:"wikipedia.com") -->
+    <string name="never_translate_site_item_list_content_description_preference">Remove %1$s</string>
+    <!-- The Delete site dialogue title will appear when the user clicks on a list item.
+             The first parameter is web site url (for example:"wikipedia.com") -->
+    <string name="never_translate_site_dialog_title_preference">Delete %1$s?</string>
+    <!-- The Delete site dialogue positive button will appear when the user clicks on a list item. The site will be deleted. -->
+    <string name="never_translate_site_dialog_confirm_delete_preference">Delete</string>
+    <!-- The Delete site dialogue negative button will appear when the user clicks on a list item. The dialog will be dismissed. -->
+    <string name="never_translate_site_dialog_cancel_preference">Cancel</string>
+
+    <!-- Download languages preference screen -->
+    <!-- Title of the download languages preference screen toolbar.-->
+    <string name="download_languages_toolbar_title_preference">Download Languages</string>
+    <!-- Screen header presenting the download language preference feature. It will appear under the toolbar.The first parameter is "Learn More," a clickable text with a link. Talkback will append this to say "Double tap to open link to learn more". -->
+    <string name="download_languages_header_preference">Download complete languages for faster translations and to translate offline. %1$s</string>
+    <!-- Clickable text from the screen header that links to a website. -->
+    <string name="download_languages_header_learn_more_preference">Learn more</string>
+    <!-- The subhead of the download language preference screen will appear above the pivot language. -->
+    <string name="download_languages_available_languages_preference">Available languages</string>
+    <!-- A text in parenthesis that will appear next to the pivot language -->
+    <string name="download_languages_default_system_language_require_preference">required</string>
```

Comment:
Should this be in parentheses, according to the comment?

Reply:
I programmatically add parentheses.

Reply:
Got it, thanks. Resolving this comment

Reply:
@iorgamgabriel: Sorry, I missed something important about this, the parenthesis should not be hardcoded, see our dev best practices [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#dont-hardcode-characters)).
We should avoid hardcoding caracters, in this case for example because we don't know if languages use another sign for a parenthesis. 
Please update strings so that they appear as so:
* LANGUAGE (required) -> `%s (required)`
* LANGUAGE (100 KB) -> `%1$s (%2$s)`

Reply:
Sorry for such a late correction!

Just closing the loop here, the string comment will be fixed over in [this PR 4899](https://github.com/mozilla-mobile/firefox-android/pull/4899) and be later substituted into `%1$s (%2$s)`.

The substitution string with the new parameters for `%1$s (%2$s)` will be introduced later (v. 123) via this [PR 4810](https://github.com/mozilla-mobile/firefox-android/pull/4810) as `download_languages_language_item_preference`.